test/e2e/utils.go

```
@@ -77,6 +78,34 @@ func prepare(t *testing.T) (*framework.TestCtx, error) {
 	return ctx, nil
 }
 
+func getJaegerOperatorImages(kubeclient kubernetes.Interface, namespace string) (map[string]string, error) {
+	imageNamesMap := make(map[string]string)
+
+	deployment, err := kubeclient.AppsV1().Deployments(namespace).Get("jaeger-operator", metav1.GetOptions{IncludeUninitialized: false})
+	if err != nil {
+		if !strings.Contains(err.Error(), "not found") {
```

Wondering why the error is just not returned?